detox/src/devices/Device.js

```diff
@@ -42,6 +43,16 @@ class Device {
     await this.client.sendUserNotification(params);
   }
 
+  async setOrientation(orientation) {
+    // orientation is 'landscape' (meaning left side portrait) or 'portrait' (non-reversed)
+    const call = invoke.call(
```

Comment:
Hey, thanks for the insight, got it working without any native extension (yay 🎉 ). Though the call you describe doesn't work, it leads to an object like

```js
{
    type: "EarlGrey",
    value: "rotateDeviceToOrientation:errorOrNil:",
    id: 5
}
```

This leads to a "target is invalid" error, which, to me, looks a bit like a "bug", right?